### PR TITLE
Added FAQ around RLS

### PR DESCRIPTION
**Pull request by aaronedell**
There are now two RLS FAQS, one talking about account IDs, the other about using QuickSight groups. @megareez @aleewhitman OR @yprikhodko  Please review for typos or internal-only content.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md [hunk 1]
```diff
@@ -260,6 +260,111 @@ Once you have applied the CUDOS_Dataset_rules S3 Dataset to all your CUDOS datas
...
+11. Click the "x" to close the "Schedule a refresh" window and again to close the "CUDOS_permissions" window.
+Add the linked_account_id Attribute to the cudos_cur Dataset
+
+12. The cudos_cur dataset does not include the linked_account_id field used to apply row-level security. From the [QuickSight Dataset console](https://us-east-1.quicksight.aws.amazon.com/sn/start/data-sets), click on the “cudos_cur” dataset and select “Edit dataset”. Note, depending on which implementation of CUDOS being used, datasets may have different naming. See Considerations below for additional details.
+13. On the left side of the page, click “Add calculated field”.
+14. Under “Name”, type “linked_account_id”. On the right side of the page, double-click the “line_item_usage_account_id” field to add the field to the calculation window. This will set our new “linked_account_id” field equal to the “line_item_usage_account_id” for each row in the cudos_cur dataset. Click “Save”.
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:310]: We can add this to CUDOS dashboard template. I think many customers will benefit from this. @megareez what do you think?

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md [hunk 2]
```diff
@@ -260,6 +260,111 @@ Once you have applied the CUDOS_Dataset_rules S3 Dataset to all your CUDOS datas
...
+
+arrow_right The use of groups allows customers to quickly rollout CUDOS to a larger set of users and simplifies permission management going forward. A customer can have 1 CUDOS dashboard with many, many groups, each seeing *only* the data relevant to them.
+
+In example below, we will demonstrate restricting CUDOS access by user group to specific AWS account ids. The process includes creating a comma separated value (CSV) file that contains AWS account ids with related user groups (ie permissions) and uploading to S3. We then associate our CUDOS datasets with the CSV file containing permissions. Lastly, we use the AWS CLI to place users into QuickSight groups that correspond with the CSV groups.
+
+### Create Your Groups and Upload to S3
+
+1. Create a CSV similar to the following, saving as CUDOS_permissions.csv. For additional detail, see [here](https://docs.aws.amazon.com/quicksight/latest/user/restrict-access-to-a-data-set-using-row-level-security.html) for example rules and how permissions are applied.
+
+		GroupName,linked_account_id
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:277]: Maybe make it a code block?

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md [hunk 3]
```diff
@@ -260,6 +260,111 @@ Once you have applied the CUDOS_Dataset_rules S3 Dataset to all your CUDOS datas
...
+		Engineering,123456789123
+
+2. Upload CUDOS_permissions.csv to an S3 bucket in the same AWS account and region where you use QuickSight.
+3. Create a manifest file, saving as CUDOS_permissions_manifest.json, with the following format.
+
+		{
+  		"entries": [
+    		{"url":"s3://REPLACE_WITH_BUCKET_NAME/CUDOS_permissions.csv", "mandatory":true},
+  		]
+		}
```
Comment (yprikhodko) [inline content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:289]: Also would be good to make as a code block

**Review by yprikhodko — COMMENTED** [review]
Looks awesome! I left few minor comments. Also it would be good to add screenshoots for the steps, but this can be done as next iteration

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md (hunk 2 above)
Comment (aaronedell) [reply content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:277]: I think as a code block no? Shows up as one for me.
Screen Shot 2021-12-16 at 7 58 05 AM

File: content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md (hunk 3 above)
Comment (aaronedell) [reply content/Cost/200_Labs/200_Cloud_Intelligence/FAQ/_index.md:289]: This also shows up as a code block for me when I host it using hugo.

**Pull request closed without merging**